src/components/forms/file-dropper/file-list-item.tsx

```diff
@@ -104,7 +104,7 @@ export const FileListItem = ({
             ))}
           </FileCategoryDropdown>
         )}
-        <RemoveButton onClick={() => onRemove(index)}>
+        <RemoveButton type="button" onClick={() => onRemove(index)}>
```

Comment:
But is the type button not implicitly given by the underlying `button`. Why does this change anything?

Reply:
Before the change:
<img width="577" alt="Bildschirm­foto 2023-03-16 um 15 22 36" src="https://user-images.githubusercontent.com/57712895/225647459-7666747e-a352-4516-8204-3a8491dd390e.png">


After the change:
<img width="669" alt="Bildschirm­foto 2023-03-16 um 15 22 57" src="https://user-images.githubusercontent.com/57712895/225647269-9a439188-fddd-4c46-9822-35442f1a1dfc.png">


Apparently `type="button"` is not set implicitly. Currently, if you upload two files on https://satellytes.com/career/senior-ux-ui-designer/ and remove the first one with the remove button, the form is submitted. With the fix this is no longer the case.